### PR TITLE
Block Editor Tools: new hooks :blowfish:

### LINKED ISSUE #631: Block Editor Tools: new hooks 🐡
### Description

As titled.

### Use Case

See pr for the hooks and their possible use cases.

### PR DESCRIPTION
**Pull request by renatonascalves**
## New Hooks

- [x] `useTerm`
- [x] `usePreviewLink`
- [x] `useCurrentTheme`

## Misc

- Bump WordPress version;

fixes #631

### CODE REVIEW

File: packages/block-editor-tools/src/hooks/index.js
Comment (mogmarsh) [inline packages/block-editor-tools/src/hooks/index.js:1]: it appears these were all alphabetical before, except for `useCurrentPostId`. I think we should move that one and put your new exports in the appropriate order, if possible.

**Review by mogmarsh — APPROVED** [review]
👍

File: .changeset/grumpy-planes-work.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@alleyinteractive/block-editor-tools": minor
+"alley-scripts-demo-plugin": minor
+---
+
+Introducing new hooks
```
Comment (srtfisher) [inline .changeset/grumpy-planes-work.md:6]: Can we get some more elaboration on this?

File: packages/block-editor-tools/src/hooks/use-current-theme/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { store } from '@wordpress/core-data';
```
Comment (srtfisher) [inline packages/block-editor-tools/src/hooks/use-current-theme/index.js:1]: Can you convert this to TS?

File: packages/block-editor-tools/src/hooks/use-term/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { store } from '@wordpress/core-data';
```
Comment (srtfisher) [inline packages/block-editor-tools/src/hooks/use-term/index.js:1]: Same here re typescript

File: .changeset/grumpy-planes-work.md (hunk 1 above)
Comment (srtfisher) [inline .changeset/grumpy-planes-work.md:6]: Can we get some more elaboration on this?

**Review by srtfisher — APPROVED** [review]

File: packages/block-editor-tools/src/hooks/use-current-theme/index.js (hunk 2 above)
Comment (renatonascalves) [reply packages/block-editor-tools/src/hooks/use-current-theme/index.js:1]: I'll leave this task to https://github.com/alleyinteractive/alley-scripts/issues/106

File: packages/block-editor-tools/src/hooks/use-term/index.js (hunk 3 above)
Comment (renatonascalves) [reply packages/block-editor-tools/src/hooks/use-term/index.js:1]: I'll leave this task to https://github.com/alleyinteractive/alley-scripts/issues/106

**Pull request merged**